modules/sso.php

```diff
@@ -970,9 +970,18 @@ static function get_user_by_wpcom_id( $wpcom_user_id ) {
 	 * @return string
 	 **/
 	public function error_msg_enable_two_step( $message ) {
-		$err = __( sprintf( 'This site requires two step authentication be enabled for your user account on WordPress.com. Please visit the <a href="%1$s" target="_blank"> Security Settings</a> page to enable two step', 'https://wordpress.com/me/security/two-step' ) , 'jetpack' );
+		$error = sprintf(
+			wp_kses(
+				__(
+					'This site requires two step authentication be enabled for your user account on WordPress.com. Please visit the <a href="%1$s" target="_blank"> Security Settings</a> page to enable two step.',
```

Comment:
This should be:
Two-Step Authentication is required to access this site. Please visit  your <a href="%1$s" target="_blank">Security Settings</a> to configure Two-step Authentication for your account.

And is there a support doc we could link to?